### PR TITLE
Header links admin part

### PR DESCRIPTION
**Pull request by Crash1212**

### CODE REVIEW

**Review by TorskyM — APPROVED** [review]

File: src/translations/header.translations.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const headerTranslations = {
+  HEADER_NOT_FOUND: 'Посилання не знайдено!',
+  HEADER_ALREADY_EXIST: 'Така посилання вже існує!'
```
[the remark this answers is not in the document]
Comment (Crash1212) [reply src/translations/header.translations.js]: fixed

File: src/components/header-form/header-form.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+              data-cy='priority'
+              className={styles.textField}
+              variant='outlined'
+              label={config.labels.header.priority}
+              value={values.priority}
+              onChange={handleChange}
+              error={touched.priority && !!errors.priority}
+            />
+            {touched.priority && errors.priority && (
+              <div className={styles.inputError}>{errors.priority}</div>
```
[the remark this answers is not in the document]
Comment (Crash1212) [reply src/components/header-form/header-form.js]: fixed

File: src/pages/header/header-details/header-details.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+const valueShape = PropTypes.shape({
+  value: PropTypes.string
+});
+HeaderDetails.propTypes = {
+  match: PropTypes.shape({
+    params: PropTypes.shape({
+      id: PropTypes.string.isRequired
+    })
+  }).isRequired,
+  header: PropTypes.shape({
```
[the remark this answers is not in the document]
Comment (Crash1212) [reply src/pages/header/header-details/header-details.js]: fixed

File: src/pages/header/header-details/header-details.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    _id: PropTypes.string,
+    priority: PropTypes.number,
+    link: PropTypes.string,
+    name: PropTypes.arrayOf(valueShape)
+  })
+};
+
+HeaderDetails.defaultProps = {
+  header: {}
```
[the remark this answers is not in the document]
Comment (Crash1212) [reply src/pages/header/header-details/header-details.js]: fixed

File: src/translations/header.translations.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Crash1212) [reply src/translations/header.translations.js]: fixed

**Review by DmytroDidukh — APPROVED** [review]

File: src/components/header-form/header-form.js [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+    validationSchema: headerValidationSchema,
+    initialValues: {
+      ukName: header.title ? header.title[0].value : '',
+      enName: header.title ? header.title[1].value : '',
+      priority: header.priority || 1,
+      link: header.link || ''
+    },
+    onSubmit: () => {
+      const newHeader = createHeader(values);
+      console.log(!!header._id);
```
Comment (TorskyM) [inline src/components/header-form/header-form.js:64]: Remove `console.log`

Comment (Crash1212) [reply src/components/header-form/header-form.js:64]: fixed

**Review by ahd3r — APPROVED** [review]

**Pull request merged**